registry/donor/views.py

```diff
@@ -172,3 +181,44 @@ def save_note():
     db.session.commit()
     flash("Poznámka uložena.", "success")
     return redirect(url_for("donor.detail", rc=note_form.rodne_cislo.data))
+
+
+@blueprint.post("/override/")
+@login_required
+def save_override():
+    override_form = DonorsOverrideForm()
+    delete = True if "delete_btn" in request.form else False
+
+    if override_form.validate_on_submit():
+        if not delete:
+            override = DonorsOverride(**override_form.field_data)
+            db.session.merge(override)
+            db.session.commit()
+
+            DonorsOverview.refresh_overview()
+            flash("Výjimka uložena", "success")
+        else:
+            override = DonorsOverride.query.get(override_form.rodne_cislo.data)
+            if override is not None:
+                db.session.delete(override)
+                db.session.commit()
+
+                DonorsOverview.refresh_overview()
+                flash("Výjimka smazána", "success")
+            else:
+                flash("Není co mazat", "warning")
+    else:
+        flash_errors(override_form)
+
+    return redirect(url_for("donor.detail", rc=override_form.rodne_cislo.data))
+
+
+@blueprint.get("/override/all")
+@login_required
+def get_overrides():
```

Comment:
Velmi zajímavá implementace. Původní myšlenka byla, že pokud má dárce aktivní override, tak se jen zvýrazní celý řádek v tabulce, ale tohle je zajímavější, i když trošku komplexnejší.

Pozor na to, že stejné zvýraznění je třeba udělat v tabulkách pro přípravu a udělování ocenění.